src/webui/api/serialize/serialize_torrent.cpp

```diff
@@ -160,6 +160,7 @@ QVariantMap serialize(const BitTorrent::Torrent &torrent)
         {KEY_TORRENT_LAST_ACTIVITY_TIME, getLastActivityTime()},
         {KEY_TORRENT_AVAILABILITY, torrent.distributedCopies()},
         {KEY_TORRENT_REANNOUNCE, torrent.nextAnnounce()},
+        {KEY_TORRENT_COMMENT, torrent.comment()},
```

Comment:
I could be wrong but it seems to me that you wouldn't need this.

Reply:
> I could be wrong but it seems to me that you wouldn't need this.

I suppose it is needed for WebUI to provide the similar feature of copy comments, no?

Reply:
> Before this PR, comment textarea in webui is already filled. So I suppose the data is already there but it needs real testing.

It is most likely requested for current torrent only (not via "sync" API).

Reply:
>I suppose it is needed for WebUI to provide the similar feature of copy comments, no?

Before this PR, comment textarea in webui is already filled. So I suppose the data is already there but it needs real testing.

Reply:
Doesn't work without it. https://github.com/qbittorrent/qBittorrent/pull/19846/files#diff-59d033a65df0cda6470d4c7770ddf459d7c9a7ce9ddec8738d846e319a404141R1016